Cilicon/Config/GitLabProvisionerConfig.swift

```diff
@@ -17,6 +17,9 @@ struct GitLabProvisionerConfig: Decodable {
     /// Only used if `downloadLatest` is set to `true`
     /// Defaults to the latest macOS binary from GitLab's S3 bucket
     let downloadURL: String
+    /// An optional advanced configuration for the GitLab Runner, will be copied to the executor as `config.toml` file
```

Comment:
If I understand this correctly we're appending the [[runners]] part to the toml file. I believe this is not sufficiently clear from the documentation / comments. People may mistakenly use a config.toml that already contains a runners sections.

Reply:
GitLab performs validation on the YAML and provides feedback if something is not right with the configuration.

Example:

>FATAL: Service run failed error=toml: line 6 (last key "runners.url"): Key 'runners.url' has already been defined.
---------- Shutting Down ----------

However, I agree that this can be explained in more detail